accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/models/HistoryEvent.java

```diff
@@ -33,16 +32,22 @@
 
     private ACCESSION mergedInto;
 
+    private ACCESSION splitInto;
+
     private LocalDateTime localDateTime;
 
     private MODEL data;
 
-    public HistoryEvent(EventType eventType, ACCESSION accession, Integer version, ACCESSION mergedInto,
+    public HistoryEvent(EventType eventType, ACCESSION accession, Integer version, ACCESSION resultInto,
```

Comment:
Not totally sure about this name, I would suggest renaming `resultInto` to something like `accessionDestination` or `destination` (which is also used in the OperationEntity)?

Reply:
updated to destinationAccession